content/launch_2-firmware-update.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Manual steps to update firmware on Launch and Launch Heavy keyboards
+description: >
+  Launch firmware before version 0.12.20 will need these manual steps to make sure they can reliably upgrade firmware with `fwupd`
+keywords:
+  - Firmware
+  - Open Firmware
+  - fwupd
+  - Launch_2
+  - Launch_heavy_1
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: software-troubleshooting
+tableOfContents: true
+---
+
+## Steps to take to update firmware
```

Comment:
I think starting this with a summary of which keyboard versions this applies to (launch_2 and launch_heavy_1), and in which circumstances this article will be necessary (when prompted to update, but the update doesn't install).